src/ProgressOnderwijsUtils/Data/ParameterizedSqlObjectMapper.cs

```diff
@@ -475,7 +476,7 @@ static TRowReader<T> CreateLoadRowMethod<T>([NotNull] Func<ParameterExpression,
             static TDelegate ConvertLambdaExpressionIntoDelegate<T, TDelegate>([NotNull] Expression<TDelegate> loadRowsLambda)
             {
                 try {
-#if NET461
+#if !NETSTANDARD2_0

```

Comment:
Dit is nog nodig?  als assemblyBuilder gewoon werkt, is dat prima toch?

Reply:
Het probleem is dat CompileToMethod niet bestaat.